lib/sles4sap_publiccloud.pm

```diff
@@ -408,13 +408,26 @@ sub stop_hana {
             rc_only => 1,
             ssh_opts => $crash_ssh_opts);
 
-        # Wait till ssh disappear
-        record_info("Wait ssh disappear start");
-        my $out = $self->{my_instance}->wait_for_ssh(timeout => 60, wait_stop => 1);
-        record_info("Wait ssh disappear end", "out:" . ($out // 'undefined'));
+        # Wait till ssh port 22 disappear
+        record_info('Wait ssh disappear', 'START');
```

Comment:
Reinventing of wheel? `wait_for_ssh()` have this included, actually it's one of the original reason why it was created.

Reply:
> Reinventing of wheel? `wait_for_ssh()` have this included, actually it's one of the original reason why it was created.

I suppose `$self->{my_instance}->wait_for_ssh(timeout => 60, wait_stop => 1)` it's not working as intended, so I guess it's OK, but I think either the PR or the commit or a comment should provide more information regarding the reasons for this change.